avaframe/in3Utils/cfgUtils.py

```diff
@@ -240,7 +240,7 @@ def compareConfig(iniFile, modName, compare, modInfo=False, toPrint=True):
     return cfg, modDict
 
 
-def writeCfgFile(avaDir, module, cfg, fileName=''):
+def writeCfgFile(avaDir, module, cfg, fileName='', filePath=''):
     """ Save configuration used to text file in Outputs as moduleName_settings.ini
         or optional in Outputs/moduleName/configurationFiles/filenName.ini
```

Comment:
See below, did we not decide to write this differently?  
I think we should clean up this function to make it easier to read and drop the moduleName_settings.ini option. 
-> ini makes it pretty clear already

Reply:
see previous comment, and here should we save it to modName.ini?